ios/FluentUI/Resources/FluentUI-ios.xcassets/dismiss-20x20.imageset/Contents.json

```diff
@@ -1,15 +1,16 @@
 {
-  "images" : [
+  "images": [
     {
-      "idiom" : "universal",
-      "filename" : "ic_fluent_dismiss_20_regular.pdf"
+      "filename": "ic_fluent_dismiss_20_filled.pdf",
+      "idiom": "universal"
     }
   ],
-  "info" : {
-    "version" : 1,
-    "author" : "xcode"
+  "info": {
+    "author": "xcode",
+    "version": 1
   },
-  "properties" : {
-    "template-rendering-intent" : "template"
+  "properties": {
+    "preserves-vector-representation": true,
```

Comment:
In generally we want to avoid this for perf reasons, do we really need it?

Reply:
We don't. This image is currently being used only on the Notification Bar which does not resize.